src/app/annotationViewer/positionLine.js

```diff
@@ -0,0 +1,117 @@
+angular
+    .module("bawApp.annotationViewer.positionLine", [])
+    .directive("positionLine", ["bawApp.unitConverter", "$window", function (unitConverter, $window) {
+
+        return {
+            restrict: "A",
+            templateUrl: "annotationViewer/positionLine.tpl.html",
+            replace: true,
+            scope: {
+                media: "<media",
+                audioData: "<audioData",
+                imageClass: "<imageClass"
+            },
+            link: function (scope, elements, attributes, controller) {
+
+                var image;
+
+                /**
+                 * Searches for the closest node to the position line element that
+                 * has the class scope.imageClass
+                 * @param element
+                 * @returns DOM element or NULL
+                 */
+                this.getImageElement = function (element, depth) {
```

Comment:
tag this for deprecation - and explain why (we don't want any DOM references from searches ideally, we should always be binding to data)